src/lib389/lib389/cli_ctl/dblib.py

```diff
@@ -234,6 +289,25 @@ def dblib_bdb2mdb(inst, log, args):
         log.error(f"Failed trying to create the directory {tmpdir} needed to store the ldif files, error: {str(e)}")
         return
 
+
+    status = get_bdb_impl_status()
+    if status is BDB_IMPL_STATUS.UNKNOWN:
+        log.warning('Unable to determine if Berkeley Database library is available. If it is not the case, the migration will fail.')
+    elif status is BDB_IMPL_STATUS.NONE:
+        log.error('Berkeley Database library is not available. Maybe 389-ds-base-bdb rpm should be installed.')
+        raise RuntimeError('Berkeley Database library is not available')
+
+if pkgs[BDB_BUNDLED_RPM] is True:
+        return BDB_IMPL_STATUS.BUNDLED
```

Comment:
Looks like the indentation is broken here.

Reply:
Good catch!  in fact this part of code copied from  get_bdb_impl_status() should be removed ....